tests/functional_tests/ffront_tests/test_past_parsing.py

```diff
@@ -266,3 +267,12 @@ def domain_format_6_program(in_field: Field[[IDim], float64]):
     assert exc_info.match("Invalid call to `domain_format_6`")
 
     assert re.search("Empty domain not allowed.", exc_info.value.__cause__.args[0]) is not None
+
+
+def test_implicit_broadcast(empty_identity_def):
```

Comment:
Looks like this is a left-over from the previous state of the PR. There is no implicit broadcast for the argument here. I think this can be safely removed.